crocket/src/main/java/com/crocket/model/DirectionLine.java

```diff
@@ -1,9 +1,11 @@
 package com.crocket.model;
 
+import com.crocket.model.entity.Ball;
+
 public class DirectionLine {
     private int degreeAngle;
-    private final double height = 100;
-    private final double width = 100;
+    private final double HEIGHT = 100;
```

Comment:
Since these are assigned in the class body and presumably will be the same for all instances of the object, I would suggest making them public and static and remove the getter.

Reply:
fixed